command/base.go

```diff
@@ -63,6 +63,7 @@ type BaseCommand struct {
 	flagOutputCurlString bool
 	flagOutputPolicy     bool
 	flagNonInteractive   bool
+	flagAddrWarning      string
```

Comment:
```suggestion
	addrWarning      string
```

Our convention is to use `flag` as a prefix for command-line flags, but this isn't one, it's just a string that we want to emit if someone calls Client.